hq_superset/views.py

```diff
@@ -1,10 +1,13 @@
 import logging
+import ast
+import pandas as pd
+import superset
+import sqlalchemy
```

Comment:
:thinking: Importing pandas and sqlalchemy in a "views" module smells dodgy. I'm not a fan of modules called "utils" cos they often degrade into a dumping ground for code that belongs in other places, but at the moment `utils.py` is kind of a service layer, where pandas and sqlalchemy feel more at home.

Reply:
:thinking: :thinking: on second thoughts, this module is already doing HQ API client stuff, and database stuff. I still think we should move those to separate modules, but maybe not to bother in this PR.

Reply:
Agreed Norman. On that note, I am the one that's mostly responsible for the not-so-good shape of the code here since this was written as an MVP, deprioritising unit tests and other standards. My idea was to refactor these things as part of Automated-tests task. If refactoring is difficult as part of that work, I am happy to create a separate ticket for refactor. Thanks for bringing this up!